extensions/ql-vscode/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-codeql",
-  "version": "1.3.11",
```

Comment:
Hm guess we missed this in the release bump.

Reply:
Darn...I should put that in my notes.